xbmc/pvr/PVRManager.cpp

```diff
@@ -138,6 +137,30 @@ CPVRManager &CPVRManager::GetInstance()
   return CServiceBroker::GetPVRManager();
 }
 
+CPVRChannelGroupsContainerPtr CPVRManager::ChannelGroups(void) const
+{
+  CSingleLock lock(m_critSection);
+  return m_channelGroups;
+}
+
+CPVRRecordingsPtr CPVRManager::Recordings(void) const
+{
+  CSingleLock lock(m_critSection);
+  return m_recordings;
+}
+
+CPVRTimersPtr CPVRManager::Timers(void) const
+{
+  CSingleLock lock(m_critSection);
+  return m_timers;
+}
+
+CPVRClientsPtr CPVRManager::Clients(void) const
+{
+  // note: m_addons is only set/reset in ctor/dtor, thus logically const. no need for a lock here.
```

Comment:
Maybe it's better to be safe than sorry in case someone changes something in the future?